contrib/forms/psychiatrySet/intakeverslag/autosave.php

```diff
@@ -23,12 +23,12 @@
 /////////////////
 // here we check to se if there was an autosave version prior to the real save 
 $vectAutosave = sqlQuery("SELECT id, autosave_flag, autosave_datetime FROM form_intakeverslag 
-                            WHERE pid = ".$_SESSION["pid"].
-                            " AND groupname='".$_SESSION["authProvider"].
-                            "' AND user='".$_SESSION["authUser"]."' AND
-                            authorized=$userauthorized AND activity=1
+                            WHERE pid = ?
```

Comment:
Highly recm not setting query strings up like this because all those extra spaces wind up being transmitted. It may look nice in source but is very inefficient for sql parser and overhead. It's okay to have a continuous string to a point, maybe 120 characters then concat.

Reply:
hi @sjpadgett , Saving the internet bandwidth one microbit at a time :)
(one thing to note here is we are not testing the changes in these old forms; ie. the less keystrokes and potential for syntax errors from our big thumbs, the better)

Reply:
Nothing worst than an angry parser that exclaims "All right already, when will I get something to parse". I started up writing machine language firmware so, bits and nanoseconds count. :)

Reply:
After doing the code review on this, realizing how painful this must of been with those large queries. Lets prioritize @d3sandoval 's sanity over the parser's sanity :)